.github/workflows/release.yml

```diff
@@ -58,3 +63,56 @@ jobs:
           message: |
             :ghost: [${{ github.repository }}] Release *${{ github.ref_name }}* didn't get triggered in Buildkite.
             Build: (<${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|here>)
+
+  post-release:
+    name: "Bump versions and create PR"
```

Comment:
As far as I see `pre-release` and `post-release differ in:
- the maven goal
- the branch name

Maybe they could be converged in a composite action to avoid duplicated code, what do you think?

Reply:
🤔 What about using a reusable workflow?
Ref: https://github.com/elastic/ecs-logging-java/pull/226/commits/531e321ec4cb7643745e39e3dd9a2c2414d18314